frameworks/TypeScript/bun/bun.dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:bookworm as build
```

Comment:
There is an official docker image which might fit better? https://hub.docker.com/r/oven/bun

Reply:
Switch